docs/cn/compute/Tensorflow.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: global
+title: 在Alluxio-FUSE上运行Tensorflow
+nickname: Tensorflow
+group: Compute Integrations
+priority: 6
+---
+本指南介绍如何在Alluxio POSIX API上运行[Tensorflow](https://www.tensorflow.org/)。
+
+* 目录
+{:toc}
+
+## 概述
+
+Tensorflow使开发人员可以快速轻松地开始使用深度学习。 
+[深度学习]({{ '/en/compute/Deep-Learning.html' | relativize_url}})部分介绍了深度学习的数据挑战 
+以及Alluxio如何帮助解决这些挑战。 
+本教程针对在Alluxio POSIX API之上运行Tensorflow提供一些动手实例和技巧。
+
+## 先决条件
+
+*安装或刷新Java为Java 8 Update 60或更高版本(8u60 +)，64位。
+* Alluxio已设置并好并正常运行。
+*已安装[Tensorflow](https://www.tensorflow.org/install/pip)。本指南使用Tensorflow **v1.15**。
+
+## 设置Alluxio POSIX API
+
+在本节中，我们将按照
+[POSIX API部分]({{ '/en/api/POSIX-API.html' | relativize_url}})中的介绍来设置Alluxio POSIX API
+和允许Tensorflow应用程序通过Alluxio POSIX API来访问数据。
+
+运行以下命令在Linux上安装FUSE
+
+```
+$ yum install fuse fuse-devel
+```
+
+在MacOS上下载[osxfuse dmg文件(https://github.com/osxfuse/osxfuse/releases/download/ osxfuse-3.8.3 / osxfuse-3.8.3.dmg)，然后按照安装说明进行操作。
+
+在Alluxio根目录下建立一个文件夹 
+
+```console
+$ ./bin/alluxio fs mkdir /training-data
+```
+
+创建一个文件夹`/ mnt / fuse`，改变文件夹的所有者为当前用户(`$(WHOAMI )`)， 
```

Comment:
remove spaces between `/mnt/fuse`. everything between backticks is thought as code instead of text. For code path, spaces are not needed